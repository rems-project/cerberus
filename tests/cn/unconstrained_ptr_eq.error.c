int f(int *p, int *q)
/*@
ensures
    return == 0i32;
@*/
{
    return p == q;
}

