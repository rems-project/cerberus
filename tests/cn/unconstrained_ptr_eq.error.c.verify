return code: 1
tests/cn/unconstrained_ptr_eq.error.c:7:12: warning: Cannot rule out ambiguous pointer equality case (addresses equal, but provenances differ)
    return p == q;
           ~~^~~~ 
[1/1]: f -- fail
tests/cn/unconstrained_ptr_eq.error.c:7:5: error: Unprovable constraint
    return p == q;
    ^~~~~~~~~~~~~~ 
Constraint from tests/cn/unconstrained_ptr_eq.error.c:4:5:
    return == 0i32;
    ^~~~~~~~~~~~~~~ 
State file: file:///tmp/state__unconstrained_ptr_eq.error.c__f.html
