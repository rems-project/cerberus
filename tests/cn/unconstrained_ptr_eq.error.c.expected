return code: 1
[1/1]: f -- fail
tests/cn/unconstrained_ptr_eq.error.c:7:5: error: Unprovable constraint
    return p == q;
    ^~~~~~~~~~~~~~ 
Constraint from tests/cn/unconstrained_ptr_eq.error.c:4:5:
    return == 0i32;
    ^~~~~~~~~~~~~~~ 
State file: file:///tmp/state__unconstrained_ptr_eq.error.c__f.html
